src/01-functions.js

```diff
@@ -74,7 +84,10 @@ function multiply() {}
  *  average(10, 6);
  *  //> 8 // 10 + 6 = 16; 16/2 = 8;
  */
-function average() {}
+function average(a, b) {
+  let sum = (a + b) / 2;
```

Comment:
this variable represents the average not the sum now so I would either change the variable to average or just return the average.